packages/block-editor/src/components/provider/use-block-sync.js

```diff
@@ -40,25 +40,25 @@ import { useRegistry } from '@wordpress/data';
  *   controllers.
```

Comment:
More examples in this file of second line descriptions not aligned with the first line